### PR TITLE
change min ios version to 12

### PR DESCRIPTION
**Pull request by billypchan**
## What's new in this PR?

drop support of iOS 10 and 11
drop arch armv7

### CODE REVIEW

**Review by KaterinaWire — APPROVED** [review]

File: Resources/Configurations/zmc-config/ios.xcconfig [hunk 1]
```diff
@@ -27,13 +27,13 @@
 SDKROOT = iphoneos
 SUPPORTED_PLATFORMS = iphoneos iphonesimulator
 
-ARCHS[sdk=iphoneos*] = arm64 armv7
```
Comment (johnxnguyen) [inline Resources/Configurations/zmc-config/ios.xcconfig:30]: Why do we drop `armv7`?

Comment (billypchan) [reply Resources/Configurations/zmc-config/ios.xcconfig:30]: `armv7` is not needed if we drop iOS10.

Comment (johnxnguyen) [reply Resources/Configurations/zmc-config/ios.xcconfig:30]: Why though? Just curious.

Comment (billypchan) [reply Resources/Configurations/zmc-config/ios.xcconfig:30]: iOS 11 only support arm64 devices, so we can drop armv7 32 devices support.

**Review by johnxnguyen — APPROVED** [review]
Don't forget that we're not going to drop support in the next public release, but the one after that.

**Comment by billypchan** [comment]
@zenkins release major

**Pull request merged**